File: landingpage/lib/legal-identity.ts
// =============================================================================
// BETREIBER-IDENTITÄT — bestätigt aus Handelsregisterauszug (HRB 78219).
// =============================================================================
// Diese Datei ist die EINZIGE Stelle der Website, an der die Anbieter-
// Identität steht. Impressum + Datenschutzerklärung lesen von hier.
//
// Mobile-App: gleiche Angaben in
//   mobile-app/lib/core/legal/legal_identity.dart
// =============================================================================

export const legalIdentity = {
  /** Firmierung bzw. voller Name des Betreibers. */
  companyName: 'TradeRiver GmbH',
  /** Ladungsfähige Anschrift — Straße + Hausnummer. */
  street: 'Kölner Straße 1',
  /** PLZ + Ort. */
  zipCity: '51379 Leverkusen',
  country: 'Deutschland',
  /** Erreichbarkeit. */
  email: 'info@ahrens-re.de',
  phone: '',
  /** Vertretungsberechtigte natürliche Person (laut Handelsregister). */
  representative: 'Jonathan Ahrens',
  representativeRole: 'Geschäftsführer',
  /** Registergericht + Registernummer — § 5 Abs. 1 Nr. 4 DDG Pflichtangabe. */
  registerInfo: 'Amtsgericht Köln, HRB 78219',
} as const;

/** Einzeiler "Name, Straße, PLZ Ort" für Fließtext (Datenschutz, § 18 MStV). */
export const legalIdentityLine = `${legalIdentity.companyName}, ${legalIdentity.street}, ${legalIdentity.zipCity}`;

File: mobile-app/lib/core/legal/legal_identity.dart
// =============================================================================
// BETREIBER-IDENTITÄT — VOR LIVE-LAUNCH AUSFÜLLEN! (Live-Blocker L1)
// =============================================================================
// Einzige Stelle der App mit der Anbieter-Identität; der Impressum-Screen
// liest von hier. Werte sind bewusst erkennbare Platzhalter.
// Website-Pendant: landingpage/lib/legal-identity.ts — beide synchron halten.
// =============================================================================

class LegalIdentity {
  static const companyName = 'TradeRiver GmbH';
  static const street = 'Kölner Straße 1';
  static const zipCity = '51379 Leverkusen';
  static const country = 'Deutschland';
  static const email = '1ahrensj@gmail.com';
  static const phone = '';
  static const representative = 'Jonathan Ahrens';
  static const representativeRole = 'Geschäftsführer';
  static const registerInfo = 'Amtsgericht Köln, HRB 78219';

  static const addressBlock =
      '$companyName\n$street\n$zipCity\n$country';
  static const identityLine = '$companyName, $street, $zipCity';
}
